js/scripts.js

```diff
@@ -35,6 +35,7 @@ var app = {
         }, 500);
 
       app.updateURL(url);
+      $('input').val('');
```

Comment:
Since we added this line do we still need this: https://github.com/adorableio/LMHT/pull/18/files#diff-549b6b2bff82a0ad6730c0f1cd665a19R8